style.css

```diff
@@ -388,21 +388,22 @@ contact me section
   width: 85%;
 }
 
```

Comment:
- [ ] Why dont you fix the horizontal overflow of your project on mobile @375 screen width to make your project responsive?

Reply:
Could please provide me more info here? I don't get it, so you are suggesting changing the with on line 388, and then adjusting all margin and padding within form elements to make it more responsive? I would appreciate it, thanks @Owusu-Desmond